### PR TITLE
Netstandard coverage and test reporting

### PR DESCRIPTION
**Pull request by baumatron**
Adds code coverage for netstandard as well as AppVeyor test reporting.

Also fixes up some formatting issues and warnings in a few files.

### CODE REVIEW

**Comment by baumatron** [comment]
Fixes #110 and #111.

File: .appveyor.yml [hunk 1]
```diff
@@ -93,17 +93,23 @@ after_build:
...
+  # Run the netstandard-specific tests and report code coverage numbers.
   # This won't likely report to the AppVeyor UI since we aren't specifying the output file/format. We might
   # need to directly use the console runner or wait for a new version of the NUnit 3 Test Adapter to support
   # the same '--result=' option as above.
   #
-  # This should cause a build to happen as well.
-  - dotnet test -c netstandard .\Keen.NetStandard.Test\Keen.NetStandard.Test.csproj
+  # Explicitly build with DebugType=Full specified. This means a non-portable, Windows-style PDB
+  # will be generated, which OpenCover currently requires. We can stop doing this once OpenCover supports the new
+  # portable PDB format.
```
Comment (masojus) [inline .appveyor.yml]: Maybe mention that the `-oldStyle` flag is also required, at least for now.

File: .appveyor.yml [hunk 2]
```diff
@@ -93,17 +93,23 @@ after_build:
...
+  - .\tools\OpenCover.4.6.519\tools\OpenCover.Console.exe -target:dotnet.exe -targetargs:"test --logger \"trx;LogFileName=netstandard_test_results.xml\" .\Keen.NetStandard.Test\Keen.NetStandard.Test.csproj /property:DebugType=Full" -mergeoutput -output:coverage.xml -filter:"+[*]* -[*.Test]*" -oldStyle -returntargetcode -register:user
 
+  # Upload merged coverage results from both test runs
+  - .\tools\coveralls.net.0.8.0-unstable0013\tools\csmacnz.Coveralls.exe --opencover -i .\coverage.xml
+
+  # Upload netstanadard test results to appveyor
```
Comment (masojus) [inline .appveyor.yml]: Typo: 'netstanadard'->'netstandard'

File: Keen.NET.Test/QueryTest.cs [hunk 3]
```diff
@@ -1138,12 +1138,12 @@ public void Serialize_SimpleValue_Success()
         {
             var filter = new QueryFilter("prop", QueryFilter.FilterOperator.Equals(), "val");
 
-            var json = JObject.FromObject(filter).ToString();
+            var json = JObject.FromObject(filter).ToString(Newtonsoft.Json.Formatting.None);
```
Comment (masojus) [inline Keen.NET.Test/QueryTest.cs]: Thanks, we really need to stop embedding strings like this for these tests, probably as part of fixing Issue #118.

File: Keen.NetStandard.Test/Keen.NetStandard.Test.csproj [hunk 4]
```diff
@@ -12,6 +12,14 @@
     <PackageReference Include="NUnit3TestAdapter" Version="3.8.0" />
   </ItemGroup>
 
+  <!--
+    Include reference to Mono.Cecil as workaroud for https://github.com/nunit/nunit3-vs-adapter/issues/325
```
Comment (masojus) [inline Keen.NetStandard.Test/Keen.NetStandard.Test.csproj]: Interesting, yeah I hadn't realized this would be necessary.

File: .appveyor.yml [hunk 5]
```diff
@@ -93,17 +93,23 @@ after_build:
 test_script:
   - nuget install OpenCover -Version 4.6.519 -OutputDirectory tools
   - nuget install coveralls.net -Version 0.8.0-unstable0013 -PreRelease -OutputDirectory tools
-  - .\tools\OpenCover.4.6.519\tools\OpenCover.Console.exe -target:nunit3-console.exe -targetargs:"--noheader .\Keen.NET.Test\bin\netfx\Keen.Net.Test.dll .\Keen.NET_35.Test\bin\netfx\Keen.NET_35.Test.dll --result=results.xml;format=AppVeyor" -filter:"+[*]* -[*.Test]*" -returntargetcode -register:user
-  - .\tools\coveralls.net.0.8.0-unstable0013\tools\csmacnz.Coveralls.exe --opencover -i .\results.xml
+  - .\tools\OpenCover.4.6.519\tools\OpenCover.Console.exe -target:nunit3-console.exe -targetargs:"--noheader .\Keen.NET.Test\bin\netfx\Keen.Net.Test.dll .\Keen.NET_35.Test\bin\netfx\Keen.NET_35.Test.dll --result=results.xml;format=AppVeyor" -output:coverage.xml -filter:"+[*]* -[*.Test]*" -returntargetcode -register:user
 
-  # While we're at it, run the netstandard-specific tests. We'll figure out code coverage for these later.
+  # Run the netstandard-specific tests and report code coverage numbers.
   # This won't likely report to the AppVeyor UI since we aren't specifying the output file/format. We might
```
Comment (masojus) [inline .appveyor.yml]: I think this part of the comments isn't valid anymore since we're doing the TRX dance and uploading manually.

**Pull request merged**